app/src/main/AndroidManifest.xml

```diff
@@ -11,6 +11,7 @@
     <uses-permission android:name="android.permission.ACCESS_WIFI_STATE" />
     <uses-permission android:name="android.permission.FOREGROUND_SERVICE" />
     <uses-permission android:name="android.permission.BLUETOOTH" />
+    <uses-feature android:name="android.hardware.type.automotive" android:required="true"/>
```

Comment:
Isn't this for the case of installation on head unit?

Reply:
> Isn't this for the case of installation on head unit?

Yes, thanks, it can be deleted.